packages/core/test/combinator/applicative-test.js

```diff
@@ -1,24 +1,23 @@
-import { spec } from 'buster'
-const { describe, it } = spec
+import { describe, it } from 'mocha'
 
 import { assertSame } from '../helper/stream-helper'
 import { ap } from '../../src/combinator/applicative'
 import { just as streamOf } from '../../src/source/core'
```

Comment:
Not strictly necessary, but since you're in this code, how about renaming `streamOf` usages to `just` so we can get rid of this `as`?